sklearn/pipeline.py

```diff
@@ -779,6 +779,9 @@ class FeatureUnion(TransformerMixin, _BaseComposition):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+        ..versionchanged:: v0.20
+           `n_jobs` change from ``1`` to ``None``
```

Comment:
```suggestion
           `n_jobs` changed from ``1`` to ``None``
```

I think it makes sense to have the descriptions in the past tense. You can check the other `versionchanged` instances to confirm.